src/current/cockroachcloud/managing-cmek.md

```diff
@@ -236,6 +248,8 @@ To rotate the CMEK keys for one or more cluster regions:
       --data "@cmek_config.json"
     ```
 
+1. When rotation is complete and the cluster's configuration has been validated, and you have verified that no other clusters are using the prior CMEK, you can optionally disable the prior CMEK or schedule it for destruction. It can be useful to check your KMS's audit logs for encryption and decryption calls to the key. Check the documentation for your KMS for for details, including how to restore a destroyed key and\ when destruction becomes permanent.
```

Comment:
```suggestion
1. When rotation is complete and the cluster's configuration has been validated, and you have verified that no other clusters are using the prior CMEK, you can optionally disable the prior CMEK or schedule it for destruction. It can be useful to check your KMS's audit logs for encryption and decryption calls to the key. Check the documentation for your KMS for details, including how to restore a destroyed key and when destruction becomes permanent.
```

Reply:
Done